Sources/NIOIMAPCore/Grammar/Date/InternalDate.swift

```diff
@@ -95,7 +95,7 @@ extension ServerMessageDate {
         /// Time zone offset in minutes.
         public let zoneMinutes: Int
 
-        /// Creates a new `Components` collection from the given parameters. Note that currently no sanity checks are performed.
+        /// Creates a new `Components` collection from the given parameters. Note that currently no language checks are performed.
```

Comment:
```suggestion
        /// Creates a new `Components` collection from the given parameters. Note that currently no soundness checks are performed.
```

Reply:
Meh, GH UI doesn't let me commit this suggestion for some reason, so I applied it in my commit.